catalog/UCSB-CHG/UCSB-CHG_CHIRPS_PENTAD.jsonnet

```diff
@@ -20,15 +20,15 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
     ee_const.ext_ver,
   ],
   id: id,
-  title: 'CHIRPS Pentad: Climate Hazards Center InfraRed Precipitation With Station Data (Version 2.0 Final)',
+  title: 'CHIRPS Precipitation Pentad: Climate Hazards Center InfraRed Precipitation With Station Data (Version 2.0 Final)',
   version: '2.0',
   'gee:type': ee_const.gee_type.image_collection,
   description: |||
     Climate Hazards Center InfraRed Precipitation with Station data (CHIRPS)
     is a 30+ year quasi-global rainfall dataset. CHIRPS incorporates
     0.05° resolution satellite imagery with in-situ station data
     to create gridded rainfall time series for trend analysis and seasonal
-    drought monitoring.
+    drought monitoring. *Note to users: A new version of CHIRPS (version 3.0) is available; please see https://chc.ucsb.edu/data and the Earth Engine catalog to transition to CHIRPS v3.0.
```

Comment:
I'd start a new para and rephrase:

*Note to users*: [A new CHRIPS version 3.0 is available](https://chc.ucsb.edu/data). This version will soon be added to the Earth Engine catalog.

Same for the other file.  (Both files can be in the same PR, BTW.)